ops/framework.py

```diff
@@ -1054,6 +1055,14 @@ def __eq__(self, other):
         else:
             return NotImplemented
 
+    def as_dict(self) -> typing.Dict:
+        """Return a copy of the backing data as a dict."""
+        return dict(self._under)
+
+    def copy(self) -> typing.Dict:
+        """Returns a copy of the backing `dict` as a deep copy dict."""
+        return dict(self._under)
```

Comment:
Both `as_dict` and `copy` now return the same thing, both shallow copy.
Is `copy` pending removal?